scripts/setup-ci

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Can this file be integrated with scripts/setup?

It would be a pity if they fell out of sync.